packages/contentstack/package.json

```diff
@@ -22,16 +22,16 @@
   },
   "dependencies": {
     "@contentstack/cli-auth": "^1.1.1",
-    "@contentstack/cli-cm-bootstrap": "^1.2.0",
+    "@contentstack/cli-cm-bootstrap": "^1.2.1",
     "@contentstack/cli-cm-bulk-publish": "^1.1.1",
-    "@contentstack/cli-cm-clone": "^1.2.0",
-    "@contentstack/cli-cm-export": "^1.3.1",
-    "@contentstack/cli-cm-export-to-csv": "^1.1.0",
-    "@contentstack/cli-cm-import": "^1.3.1",
-    "@contentstack/cli-cm-migrate-rte": "^1.2.0",
-    "@contentstack/cli-cm-seed": "^1.2.0",
-    "@contentstack/cli-command": "^1.1.0",
-    "@contentstack/cli-config": "^1.1.0",
+    "@contentstack/cli-cm-clone": "^1.2.1",
+    "@contentstack/cli-cm-export": "^1.3.2",
```

Comment:
@abhinav-from-contentstack Generate new plugin versions for this entire section after accommodating the above review comments.